lib/Config/LocationAwareConfigRepository.php

```diff
@@ -227,9 +235,8 @@ public function isWriteable(?string $key = null, ?string $dataSource = null): bo
      */
     public function getWriteTarget(): string
     {
-        $env = $this->writeTargetEnvVariableName ? $_SERVER[$this->writeTargetEnvVariableName] ?? null : null;
-        if ($env) {
-            $writeLocation = $env;
+        if ($this->writeTarget) {
+            $writeLocation = $this->writeTarget;
         } else {
             $writeLocation = $this->defaultWriteLocation;
         }
```

Comment:
It could even be written like this, right?

```suggestion
        $writeLocation = $this->writeTarget ?? $this->defaultWriteLocation;
```